src/main/java/com/uid2/admin/Main.java

```diff
@@ -206,6 +206,14 @@ public void run() {
         }
     }
 
+    private TaggableCloudStorage createStorage() {
```

Comment:
Should this be a util function instead? Since it's used here and in PrivateSiteDataSyncJob

Reply:
It's temporary for backwards compatibility. The idea is to delete it once we're back to one auth method.